acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -1,5 +1,5 @@
-step "Install PC1 repository" do
-  install_puppetlabs_release_repo(hosts, 'pc1')
+step "Install puppet repository" do
+  install_puppetlabs_release_repo(hosts, 'puppet')
```

Comment:
https://github.com/puppetlabs/beaker-puppet/pull/9 will address this change and allow beaker to set up the new puppet5 repos